eclipse-collections/src/main/java/org/eclipse/collections/impl/tuple/primitive/PrimitiveTuples.java

```diff
@@ -91,6 +91,9 @@
 import org.eclipse.collections.api.tuple.primitive.ShortObjectPair;
 import org.eclipse.collections.api.tuple.primitive.ShortShortPair;
 
+/**
+ * This class should be used to create instances of object/primitive, primitive/primitive and primitive/primitive pairs.
```

Comment:
This looks wrong: last two are the same, maybe it was supposed to be primitive/object?

Reply:
good catch.  fixed.